City-of-Helsinki/drupal-helfi-etusivu
# UHF-9964

# [UHF-9964](https://helsinkisolutionoffice.atlassian.net/browse/UHF-9964)
Added fields to news item
 - show in recommendations
 - show recommendations block

## What was done
Added new fields, added the fields to be used within the recommendations workflow

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout UHF-9964`
  * `make fresh`
* Run `make drush-cr`

## How to test
Check the previous testing instructions
- run `drush:helfi:annif-entity-defaults`
- run `drush cr`
- Enable the recommendations block
- Go check some news items recommendations
- Based on what recommendations you see, go edit one of them and change the "show in recommendations" -value and save the node.
- run drush cron
- The recommendation should disappear from recommendation blocks
- go to edit a news item and disable the block by changing the "show recommendations block" -value


## Continuous documentation



## Translations



## Other PRs


* Link to other PR


[UHF-9964]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-9964?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
